common/authorization/authorizer.go

```diff
@@ -55,7 +54,7 @@ type (
 
 // Authorizer is an interface for authorization
 type Authorizer interface {
-	Authorize(ctx context.Context, attributes *Attributes) (Result, error)
+	Authorize(ctx context.Context, caller *Claims, target *CallTarget) (Result, error)
```

Comment:
can you change from `*Claims` (pointer to struct) to interface?
same as `*CallTarget`

Reply:
Discussed offline. No need to turn these structs into interfaces.